rootex/core/audio/audio_source.h

```diff
@@ -36,6 +48,12 @@ class AudioSource
 	ALuint getSourceID() const;
 	/// Get audio duration in seconds.
 	virtual float getDuration() const = 0;
+
+	void setPosition(Vector3 position);
+	void setModel(AttenuationModel model = AttenuationModel::linear);
+	void setRolloffFactor(ALfloat rolloffFactor);
+	void setReferenceDistance(ALfloat referenceDistance);
```

Comment:
setRollofFactor() and setReferenceDistance() need doc strings. It is not apparent what they affect